maestro/common/subflow-skip-chrome-welcome.yaml

```diff
@@ -0,0 +1,10 @@
+appId: com.stripe.android.financialconnections.example
```

Comment:
Is there a way so that the appId is not needed here?

Reply:
Unfortunately it's required, but it can be passed via params I think! looking. 

<img width="1237" alt="image" src="https://user-images.githubusercontent.com/99293320/207729315-59e49f30-8a64-4f7f-b31e-177c5aac7da2.png">

Reply:
https://github.com/stripe/stripe-android/pull/5953/commits/aaa093636a2b09b37381b21cf77ba6442abbac16